sympy/core/tests/test_subs.py

```diff
@@ -862,3 +862,13 @@ def test_issue_19558():
 
     assert e.subs(x, oo) == AccumBounds(-oo, oo)
     assert (sin(x) + cos(x)).subs(x, oo) == AccumBounds(-2, 2)
+
+def test_subs_undefined_result():
```

Comment:
Can you use two blank lines between test functions (also fix this for the function above)?